cmd/cluster/checkbanneduser.go

```diff
@@ -0,0 +1,69 @@
+package cluster
+
+import (
+	"fmt"
+	"github.com/openshift/osdctl/pkg/utils"
+	"github.com/spf13/cobra"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+)
+
+const BanCodeExportControlCompliance = "export_control_compliance"
+
+func newCmdBannedUser() *cobra.Command {
+	return &cobra.Command{
+		Use:               "check-banned-user",
+		Short:             "Checks if the cluster owner is a banned user.",
+		Args:              cobra.ExactArgs(1),
+		DisableAutoGenTag: true,
+		Run: func(cmd *cobra.Command, args []string) {
```

Comment:
Should this be `RunE:`?  

`func CheckBannedUser` returns an error, but `Run:` just drops it on the floor, so there's no indication if there was an error or not.

Reply:
I'm not sure honestly. All over the rest of the code base, I see `Run` being used with `cmdutil.CheckErr` wrapping the handler function. Tracing down, `cmdutil.CheckErr` will eventually call the following function, which seems to write to stderr and exit with a non-0 code.

```golang
func fatal(msg string, code int) {
	// nolint:logcheck // Not using the result of klog.V(99) inside the if
	// branch is okay, we just use it to determine how to terminate.
	if klog.V(99).Enabled() {
		klog.FatalDepth(2, msg)
	}
	if len(msg) > 0 {
		// add newline if needed
		if !strings.HasSuffix(msg, "\n") {
			msg += "\n"
		}
		fmt.Fprint(os.Stderr, msg)
	}
	os.Exit(code)
}
```

Reply:
Ah, good catch.  Carry on.  Nothing to see here.